app/src/main/res/values/strings.xml

```diff
@@ -343,6 +343,7 @@
   <string name="overduelist_diagnosis_label">Dx</string>
   <string name="overduelist_visited">Visited</string>
   <string name="overduelist_high_risk">High risk</string>
+  <string name="overduelist_transferred">Transferred to</string>
```

Comment:
The PRD and the label have different designs. Were the designs updated but not added to the PRD?

Reply:
I am following the designs from [Figma](https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-Production?node-id=7573%3A24915).